test/integration/narrow/routes/endemics/which-species-ms.test.js

```diff
@@ -30,56 +30,79 @@ describe('Endemics which species test', () => {
     crumb = await getCrumbs(global.__SERVER__)
   })
 
-  test('Sheep should be selected when typeOfLivestock is sheep', async () => {
+  describe('GET claim/endemics/which-species', () => {
```

Comment:
Do we have tests for how the page renders when MS is enabled, and how it renders when its not enabled?